runtime/gc_glue_java/ConfigurationDelegate.hpp

```diff
@@ -344,7 +345,18 @@ class MM_ConfigurationDelegate
 	{
 		_maximumDefaultNumberOfGCThreads = maxGCThreads;
 	}
-
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	void checkPointGCThreadCountWarningCheck(MM_EnvironmentBase *env)
+	{
+		MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
+		if (!extensions->userSpecifiedParameters._checkpointGCThreads._wasSpecified) {
+			extensions->checkpointGCthreadCount = OMR_MIN(extensions->checkpointGCthreadCount, extensions->gcThreadCount);
```

Comment:
The name of the function suggests "warning check", but it also sets this value. Can we change function name to reflect this? Or, alternatively, set value somewhere else (not sure it is better solution).

Reply:
Ok, this is just silent correction, may be we can keep it as-is

Reply:
we could consider moving the check and adjust part to OMR, and leave here only NLS print part, but I probably prefer this way, since checkpoint awareness is already mostly on J9 side.

Reply:
perhaps checkPointGCThreadCountValidation (or Verification)? but still does not suggest the methods may adjust the value... so we can append 'adjust' in the name: checkPointGCThreadCountVerifyAndAdjust or some other variation